src/controller/CHIPDeviceController.cpp

```diff
@@ -1227,6 +1227,8 @@ CHIP_ERROR DeviceCommissioner::ProcessCSR(DeviceProxy * proxy, const ByteSpan &
         mOperationalCredentialsDelegate->SetFabricIdForNextNOCRequest(mFabricInfo->GetFabricId());
     }
 
+    mOperationalCredentialsDelegate->SetCATForNextNOCRequest(kTestDeviceCAT);
```

Comment:
Wait, why would we do this unconditionally, even in normal operation?  This does not look right to me...

Reply:
Agree.  This type of thing, if it's going to be done, needs to be done from chip-tool.

Reply:
Yes, thanks for suggestion. I agree and moved it to the application layer.